src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java

```diff
@@ -126,10 +127,12 @@ public boolean apply(GHEventsSubscriber subscriber) {
             @Override
             public Void apply(GHEventsSubscriber subscriber) {
                 try {
-                    subscriber.onEvent(event, payload);
+                    notNull(subscriber).onEvent(event, payload);
+                } catch (Error error) {
+                    throw error;
                 } catch (Throwable t) {
-                    LOGGER.error("Subscriber {} failed to process {} hook, skipping...", 
-                            subscriber.getClass().getName(), event, t);
+                    LOGGER.error("Subscriber {} failed to process {} hook, skipping...",
+                           subscriber.getClass().getName(), event, t);
```

Comment:
This can cause yet another `NullPointerException` as subscriber is possibly `null` if `t` is a `NullPointerException` and in this case nothing will be logged :-(